src/analysis/normed_space/operator_norm.lean

```diff
@@ -353,6 +353,70 @@ begin
       by rwa [mul_one] }
 end
 
+section uniformly_extend
+
+variables [complete_space F] {e : E →L[𝕜] G}
+
+section
+variables (h_e : uniform_inducing e) (h_dense : dense_range e)
+
+/-- Extension of a continuous linear map `f : E →L[𝕜] F`, with `E` a normed space and `F` a complete
+    normed space, along a uniform and dense embedding `e : E →L[𝕜] G`.  -/
+def extend : G →L[𝕜] F :=
+/- extension of `f` is continuous -/
+have cont : _ := (uniform_continuous_uniformly_extend h_e h_dense f.uniform_continuous).continuous,
```

Comment:
```suggestion
have cont := (uniform_continuous_uniformly_extend h_e h_dense f.uniform_continuous).continuous,
```

Reply:
This will give an error. I think it is only allowed inside of tactic mode.

Reply:
Ah yes, I am too used to tactic mode!